Scripts/Python/ki/xKIChat.py

```diff
@@ -664,16 +664,26 @@ def AddChatLine(self, player, message, cFlags, forceKI=True):
         self.ResetFadeState()
 
     ## Display a status message to the player (or players if net-propagated).
-    def DisplayStatusMessage(self, statusMessage, netPropagate=0):
+    def DisplayStatusMessage(self, message, netPropagate=0, isMessageLocKey=False):
```

Comment:
I think it would be better to, instead of having an explicit flag `isMessageLocKey` to instead allow `message` to be a [`NamedTuple`](https://docs.python.org/3.8/library/typing.html#typing.NamedTuple), eg:
```python
class LocKey(NamedTuple):
    message: str
    pronoun: str = "KI.EmoteStrings.Their"
```

You could then use `isinstance(message, LocKey)` to implicitly decide if the message is a loc key message.